app/uk/gov/nationalarchives/omega/editorial/controllers/EditSetRecordController.scala

```diff
@@ -134,9 +135,10 @@ class EditSetRecordController @Inject() (
       val heading: String =
         resolvedMessage(MessageKeys.heading, editSetRecords.getEditSetRecordByOCI(oci).get.ccr)
       val message: String = resolvedMessage(MessageKeys.buttonDiscard)
+      val recordType: Option[RecordType] = editSetRecords.getEditSetRecordByOCI(oci).get.recordType
```

Comment:
There's a `get` here - how do we handle the case where `getEditSetRecordByOCI` is None?

Reply:
Tests added to reflect the behaviour without record type suffix.